src/taipy/gui/gui.py

```diff
@@ -280,7 +280,9 @@ def __init__(
         if css_file is None:
             script_file = pathlib.Path(self.__frame.f_code.co_filename or ".").resolve()
             if script_file.with_suffix(".css").exists():
-                css_file = f'{script_file.stem or "Taipy"}.css'
+                css_file = f"{script_file.stem}.css"
+            elif script_file.is_dir() and (script_file / "taipy.css").exists():
```

Comment:
Will you let me know how to document this case?

Reply:
I guess that when in a notebook script_file => "." (need to be confirmed)
In this case (and in every case where co_filename is None), we look for a taipy.css file in the current directory...
Not sure what current directory means in a notebook but we can find out ...